### ISSUE TITLE
Bug: Cluster images flickering on clicking events.

### ISSUE THREAD

**Issue by akc4** [diegoazh/gmap-vue#137]
**Describe the bug**
I am using Clustering on my markers and I have noticed that on any update event on my page, the images of the cluster will "blink/flicker". Either I click on a button, change the range on a range input, etc.

**To Reproduce**

I think the problem is coming from the updated method in cluster.js component.
```
 updated() {
    if (this.$clusterObject) {
      this.$clusterObject.repaint();
    }
  },
```

**Expected behavior**
Expect the cluster images not to flicker/blink on any update / click on the page.

**Current behavior**
Flickering/Blinking on any update / click

**Screenshots**

**Desktop (please complete the following information):**
 - Browser [chrome]

## Versions

- Node:
- NPM:
- Yarn:

## Package manager

- [x ] NPM
- [ ] Yarn

## Plugin version

- [ x] 1.4.x
- [ ] 1.2.x
- [ ] 1.0.x

**Comment by diegoazh** [comment]
@akc4 please don't remove sections of the issue template

**Comment by diegoazh** [comment]
Try the new version 2.0.0 and let me know if the issue pressists

**Comment by diegoazh** [comment]
@akc4 the cluster component was fixed in v2.0.3, please test it and let me know if something was wrong. Regards.
